mapping_project/lib/myhomepage.dart

```diff
@@ -14,13 +15,27 @@ class MyHomePage extends StatefulWidget {
 class _MyHomePageState extends State<MyHomePage>
     with SingleTickerProviderStateMixin {
   var maps = [
-    'assets/images/SPSMAP_Top.svg',
-    'assets/images/SPSMAP_3rd.svg',
-    'assets/images/SPSMAP_2nd.svg',
-    'assets/images/SPSMAP_1st.svg',
-    'assets/images/SPSMAP_Ground.svg'
+    'assets/images/Whole.svg',
+    'assets/images/GTB1-G.svg',
+    'assets/images/GTB1-1.svg',
+    'assets/images/GTB1-2.svg',
+    'assets/images/GTB2-G.svg',
+    'assets/images/GTB2-1.svg',
+    'assets/images/GTB2-2.svg',
+    'assets/images/Milton-G.svg',
+    'assets/images/Milton-1.svg',
+    'assets/images/Milton-2.svg',
+    'assets/images/Milton-3.svg',
+    'assets/images/Music-G.svg',
+    'assets/images/Music-1.svg',
+    'assets/images/Pepys-G.svg',
+    'assets/images/Pepys-1.svg',
+    'assets/images/Science-G.svg',
+    'assets/images/Science-1.svg',
+    'assets/images/Science-2.svg',
+    'assets/images/Science-3.svg',
   ];
-  var _moved = false;
+  //var _moved = false;
```

Comment:
why is this commented?

Reply:
This.... Shouldn't be commented - shall fix